### PR TITLE
first pass

### PR DESCRIPTION
**Pull request by lecorguille**
It's not so easy.
Lot of those terms are now commonly used around scientists.

@dannon Should we also look at the commented lines?

### CODE REVIEW

**Comment by lecorguille** [comment]
Ping @bebatut @yvanlebras

**Comment by yvanlebras** [comment]
ouh yeah!

File: client/galaxy/scripts/nls/fr/locale.js [hunk 1]
```diff
@@ -6,61 +6,61 @@ define({
...
+    "Visualisations",
 "Pages":
-    false,
+    "Pages",
 "Visualization":
-    "Représentation des données",
+    "Visualisation",
 "New Track Browser":
-    false,
+    "Nouveau Navigateur de Tracks/Pistes",
```
Comment (yvanlebras) [inline client/galaxy/scripts/nls/fr/locale.js]: Not easy to translate... Maybe something like "Nouvelle visualisation de génome" or as it is through Trackster something like "Nouvelle visualisation Trackster"...

File: client/galaxy/scripts/nls/fr/locale.js [hunk 2]
```diff
@@ -6,61 +6,61 @@ define({
...
 "Saved Visualizations":
-    false,
+    "Visualisations sauvegardés",
 "Interactive Environments":
-    false,
+    "Environnements interactifs",
```
Comment (yvanlebras) [inline client/galaxy/scripts/nls/fr/locale.js]: Not sure we have to make a direct word by word translation....  but I don't have better proposition.

File: client/galaxy/scripts/nls/fr/locale.js [hunk 3]
```diff
@@ -6,61 +6,61 @@ define({
...
+    "Recherche",
 "Mailing Lists":
-    false,
+    "Liste de diffusion",
 "Videos":
-    false,
+    "Vidéos",
 "Wiki":
-    false,
+    "Documentation",
```
Comment (yvanlebras) [inline client/galaxy/scripts/nls/fr/locale.js]: Or let "Wiki" as it's the same in french. In the meantime, I agree that maybe the link give access to "Documentation" more than ""just"" a "wiki"...

File: client/galaxy/scripts/nls/fr/locale.js [hunk 4]
```diff
@@ -6,61 +6,61 @@ define({
...
 "How to Cite Galaxy":
-    false,
+    "Comment citer Galaxy",
 "Interactive Tours":
-    false,
+    "Guides interactifs",
```
Comment (yvanlebras) [inline client/galaxy/scripts/nls/fr/locale.js]: Maybe something around "tutoriels intéractifs" or "tutoriels pas à pas" can be more illustrative... ?

File: client/galaxy/scripts/nls/fr/locale.js [hunk 5]
```diff
@@ -6,61 +6,61 @@ define({
...
     "Utilisateur",
 "Login":
-    false,
+    "Authentification",
 "Register":
-    false,
+    "Enregistrement",
 "Logged in as":
-    false,
+    "Authentification en tant que",
```
Comment (yvanlebras) [inline client/galaxy/scripts/nls/fr/locale.js]: Maybe "Authentifié en tant que" or "Authentifié comme" ?

File: client/galaxy/scripts/nls/fr/locale.js [hunk 6]
```diff
@@ -6,61 +6,61 @@ define({
...
 "Custom Builds":
-    false,
+    "Genome Builds personalisés",
```
Comment (yvanlebras) [inline client/galaxy/scripts/nls/fr/locale.js]: Maybe "Mes génomes de références" can be more illustrative ?

File: client/galaxy/scripts/nls/fr/locale.js [hunk 7]
```diff
@@ -6,61 +6,61 @@ define({
...
 "Logout":
-    false,
+    "Déconnection",
 "Saved Histories":
-    false,
+    "Historiques sauvegardés",
 "Saved Datasets":
-    false,
+    "Données sauvegardés",
```
Comment (yvanlebras) [inline client/galaxy/scripts/nls/fr/locale.js]: "Jeux de données sauvegardés"

File: client/galaxy/scripts/nls/fr/locale.js [hunk 8]
```diff
@@ -69,26 +69,26 @@ define({
 "No matching datasets found" :
     "Aucunes données correspondantes trouvées",
```
Comment (yvanlebras) [inline client/galaxy/scripts/nls/fr/locale.js]: Not sure I remember when we have this kind of message... "Aucun jeux de données appropriés trouvés"

File: client/galaxy/scripts/nls/fr/locale.js [hunk 9]
```diff
@@ -69,26 +69,26 @@ define({
...
 "An error occurred while getting updates from the server" :
-    "Une erreur s'est produite lors de l'envoi des mises à jour depuis le serveur",
+    "Une erreur s'est produite lors de la réception des données depuis le serveur",
 "Please contact a Galaxy administrator if the problem persists" :
-    "Veuillez contacter un administrateur Galaxy si ce problème persiste.",
+    "Veuillez contacter un administrateur de l'instance Galaxy si ce problème persiste.",
```
Comment (yvanlebras) [inline client/galaxy/scripts/nls/fr/locale.js]: Maybe without the "." at the end ;)

File: client/galaxy/scripts/nls/fr/locale.js [hunk 10]
```diff
@@ -69,26 +69,26 @@ define({
...
 //TODO:
 //"An error was encountered while <% where %>" :
 //false,
 "Search datasets" :
-    "Recherche dans l'ensemble des données",
+    "Rechercher des données",
```
Comment (yvanlebras) [inline client/galaxy/scripts/nls/fr/locale.js]: "Rechercher des jeux de données"

File: client/galaxy/scripts/nls/fr/locale.js [hunk 11]
```diff
@@ -69,26 +69,26 @@ define({
...
 "You are currently viewing a deleted history!" :
     "Vous consultez actuellement un historique supprimé!",
 "You are over your disk quota" :
-    "Vous avez dépassé votre quota disque",
+    "Vous avez dépassé votre quota d'espace disque",
 "Tool execution is on hold until your disk usage drops below your allocated quota" :
     "L'exécution de l'outil est en attente tant que votre utilisation d'espace disque n'est pas inférieur à votre quota attribué",
```
Comment (yvanlebras) [inline client/galaxy/scripts/nls/fr/locale.js]: Maybe something like "L'exécution de l'outil est en attente tant que votre utilisation d'espace disque dépasse le quota attribué" ?

File: client/galaxy/scripts/nls/fr/locale.js [hunk 12]
```diff
@@ -69,26 +69,26 @@ define({
...
 "All" :
     "Tout",
 "None" :
     "Aucun",
 "For all selected" :
-    "Pour tous les",
+    "Pour tous les selectionnés",
```
Comment (yvanlebras) [inline client/galaxy/scripts/nls/fr/locale.js]: Maybe something like "Pour la sélection entière" / "Pour toute la sélection" will be more french?

**Comment by yvanlebras** [comment]
@lecorguille I'm working on the rest of the file (on the commented lines)...

**Comment by dannon** [comment]
Awesome, @lecorguille, @yvanlebras!  Yes, we'll want to do the commented lines, too.  Would you want to keep working on this branch, or is it ready to merge in and then issue another PR for future work?

File: client/galaxy/scripts/nls/fr/locale.js (hunk 3 above)
Comment (lecorguille) [inline client/galaxy/scripts/nls/fr/locale.js]: It's no longer a wiki :)

File: client/galaxy/scripts/nls/fr/locale.js (hunk 6 above)
Comment (lecorguille) [inline client/galaxy/scripts/nls/fr/locale.js]: They will met the term Builds in the different tool form. I think we should not bypass the hard code Galaxy vocabulary.

File: client/galaxy/scripts/nls/fr/locale.js (hunk 7 above)
Comment (lecorguille) [inline client/galaxy/scripts/nls/fr/locale.js]: I though about that but I wanted to keep the save term for all mention of `Datasets` and it's a little long `Jeux de données`

File: client/galaxy/scripts/nls/fr/locale.js (hunk 8 above)
Comment (lecorguille) [inline client/galaxy/scripts/nls/fr/locale.js]: When you use the searching box at the top of the history panel

File: client/galaxy/scripts/nls/fr/locale.js (hunk 10 above)
Comment (lecorguille) [inline client/galaxy/scripts/nls/fr/locale.js]: Same

File: client/galaxy/scripts/nls/fr/locale.js (hunk 12 above)
Comment (lecorguille) [inline client/galaxy/scripts/nls/fr/locale.js]: Agree

File: client/galaxy/scripts/nls/fr/locale.js (hunk 1 above)
Comment (lecorguille) [inline client/galaxy/scripts/nls/fr/locale.js]: Agree for `Nouvelle visualisation de génome` it's clearer but it's not literally a translation :/

File: client/galaxy/scripts/nls/fr/locale.js (hunk 9 above)
Comment (lecorguille) [inline client/galaxy/scripts/nls/fr/locale.js]: Agree

File: client/galaxy/scripts/nls/fr/locale.js (hunk 11 above)
Comment (lecorguille) [inline client/galaxy/scripts/nls/fr/locale.js]: Ouhai

**Comment by lecorguille** [comment]
@yvanlebras 
- About the translation `Datasets` in `Jeux de données` or `Données`. You can decide if you want but for me, we need to keep the final term everywhere.

@yvanlebras are you working from me fork?

**Pull request merged**